packages/extension-koni-base/src/api/bonding/paraChain.ts

```diff
@@ -64,7 +64,7 @@ export async function getParaBondingBasics (networkKey: string, dotSamaApi: ApiP
 
   const inflationConfig = _inflation.toHuman() as unknown as InflationConfig;
   const currentInflation = getParaCurrentInflation(totalStake, inflationConfig);
-  const rewardPool = currentInflation * PARACHAIN_INFLATION_DISTRIBUTION[networkKey].reward;
+  const rewardPool = currentInflation * (PARACHAIN_INFLATION_DISTRIBUTION[networkKey].reward || PARACHAIN_INFLATION_DISTRIBUTION.default.reward);
 
```

Comment:
Need check if PARACHAIN_INFLATION_DISTRIBUTION[networkKey] exists, not PARACHAIN_INFLATION_DISTRIBUTION[networkKey].reward

Reply:
Resolved in #712